### ISSUE TITLE
[question] how to interpret Reports overview?

### ISSUE THREAD

**Issue by zifnabbe** [colloqi/piSignage#76]
Hi,

How to interpret the following table?

![image](https://user-images.githubusercontent.com/45363336/52940221-4ba07b00-3366-11e9-8d7d-b70b7dcabf95.png)

What does '1' and '0' mean?
I don't understand the difference in '1' till 18/2 08:00AM and '0' from 18/2 09:00 AM

The player isn't playing since 10:30 PM (17/2), should start again at 11:00 AM (18/2).  
So why the difference?

Thanks in advance,

Regards, 
Tom.

**Comment by colloqi** [comment]
Hi

Server-connected means player was in communication with server and playerUptime means player was "ON". 1 means true and 0 means false.

There are other events/reports which report about playlist play time

**Comment by zifnabbe** [comment]
yes, but I don't understand the reason why the player was 'off' at 9 and 10 AM, while it all hours before it was 'on'.  
The player was not scheduled from 10:30 PM till 11.00 AM. So I would thought its was 'off' from 10:30 PM till 11.00 AM, and from then 'on'. So why was the player of from '9' AM and not before?
